test/unit/presenters/formats/local_transaction_presenter_test.rb

```diff
@@ -23,6 +23,15 @@ def edition
       introduction: "hello",
       more_information: "more info",
       need_to_know: "for your eyes only",
+      scotland_availability: FactoryBot.build(
+        :devolved_administration_availability,
+      ),
+      wales_availability: FactoryBot.build(
+        :devolved_administration_availability,
+      ),
+      northern_ireland_availability: FactoryBot.build(
+        :devolved_administration_availability,
+      ),
```

Comment:
```suggestion
```

I don't think you need any of these lines because of the autobuild attribute.

Reply:
Thanks, I updated.